lib/services/file_transfer_service.dart

```diff
@@ -212,10 +212,11 @@ class FileTransferService {
         uploadedBytes += chunk.length;
         var percent = (uploadedBytes / fileLength) * 100;
         fileUploadProvider.updateSentFileTransferProgress =
-            FileTransferProgress(FileState.upload, percent, fileName);
+            FileTransferProgress(
+                FileState.upload, fileLength.toDouble(), fileName);
```

Comment:
Can we create another field for fileLength in FIleTransferProgress model ?

Reply:
okay